src/context/graphql/data.test.ts

```diff
@@ -3,7 +3,7 @@
  */
```

Comment:
I think we should preserve the flat directory structure of `src/context`. The idea is that directories resemble the API you're using, and you are still using `ctx.data` and `ctx.extensions`. I know it makes sense to group GraphQL-specific utilities respectively, but there's little downside of not doing so.

Reply:
Hey Kettanaito, I agree with you here to follow the API layout, the only reason I changed it was due to creating the generalised type `GraphQLPayloadContext`. In this folder structure, should I rename this file or move it to a different folder to be clear that it is used for Graphql data structures?

Reply:
I've moved the type to `src/typeUtils.ts`, I think that's be the right place.